sdk/servicebus/service-bus/src/core/messageReceiver.ts

```diff
@@ -1048,6 +1046,30 @@ export class MessageReceiver extends LinkEntity {
         delivery.reject(options.error || {});
       }
     });
+
+    const config: RetryConfig<any> = {
+      operation: () => {
+        return settleMessagePromise;
+      },
+      connectionId: this._context.namespace.connectionId,
+      operationType: RetryOperationType.management,
```

Comment:
The operation type is not management.
You will need to add a new entry to the enum list for the message settlement operation.

Reply:
Filed https://github.com/Azure/azure-sdk-for-js/issues/6147 to update amqp-common and update this later as it only affects logging